python/GafferUI/PathListingWidget.py

```diff
@@ -764,6 +768,8 @@ def __dragBegin( self, widget, event ) :
 			return IECore.StringVectorData( selection[0].paths() )
 
 		index = self.__indexAt( event.line.p0 )
+		self.__dragBeginIndex = index
```

Comment:
Should we only be setting this in the conditional just before `return value`? When we `return None` we haven't initiated a drag.

Reply:
Updated in ea1783d7256b60b9d2c53e41f9ae72856cd6b003.